src/digitalmarketplace/components/alert/_alert.scss

```diff
@@ -0,0 +1,27 @@
+@import "../../../govuk-frontend/tools/all";
+@import "../../../govuk-frontend/helpers/all";
+
+@include govuk-exports("digitalmarketplace/component/alert") {
+
+    .dm-alert {
+        @include govuk-responsive-padding(4);
+        @include govuk-responsive-margin(8, "bottom");
+        border: govuk-spacing(1) solid;
+    }
+    .dm-alert--success {
+        border-color: govuk-colour("green");
+    }
+    .dm-alert--error {
+        border-color: $govuk-error-colour;
+    }
+    .dm-alert--noticees {
+        border-color: govuk-colour("blue");
+    }
+    .dm-alert__title {
+        margin: 0px;
```

Comment:
we need to set the font size and font family to be consistent. To do this we need to call  `@include govuk-font(19, $weight: bold);`

Reply:
So the only difference between body and title is the font-weight, right?